DER/DerivativesContracts/Forwards.rdf

```diff
@@ -432,58 +370,38 @@
 		<skos:definition xml:lang="en">&apos;A classification of the type of fee or additional payment, e.g. brokerage, upfront fee etc. FpML does not define domain values for this element.&apos;</skos:definition>
 	</owl:DatatypeProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;settlementAmount">
 		<rdfs:label xml:lang="en">settlement amount</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardCashSettlementTermsSet"/>
+		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardCashSettlementCommitment"/>
 		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
 	</owl:ObjectProperty>
 	
 	<owl:DatatypeProperty rdf:about="&fibo-der-der-fwd;settlementDate">
 		<rdfs:label xml:lang="en">settlement date</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardCashSettlementTermsSet"/>
+		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardCashSettlementCommitment"/>
 		<skos:definition xml:lang="en">FpML = valueDate Definition for &apos;valueDate&apos; in FpML &apos;The date on which both currencies traded will settle.&apos; Definition for &apos;currency2ValueDate&apos; in FpML &apos;The date on which the currency2 amount will be settled. To be used in a split value date scenario.&apos; Notes 20 Jan: This is the scenario where &quot;t&quot; for one leg is different to &quot;t&quot; for the other leg.</skos:definition>
 	</owl:DatatypeProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;side">
-		<rdfs:label xml:lang="en">side</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardContract"/>
-		<rdfs:range rdf:resource="&fibo-der-der-fwd;ForwardCashSettlementTermsSet"/>
-	</owl:ObjectProperty>
-	
-	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;side.1">
-		<rdfs:label xml:lang="en">side</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardContract"/>
-		<rdfs:range rdf:resource="&fibo-der-der-fwd;ForwardDeliveryTermsSet"/>
-	</owl:ObjectProperty>
-	
-	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;specifies">
-		<rdfs:label xml:lang="en">specifies</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardValuationTermsSet"/>
-		<rdfs:range rdf:resource="&fibo-der-der-fwd;UnderlyingValuation"/>
-	</owl:ObjectProperty>
-	
 	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;undertakenBy">
 		<rdfs:label xml:lang="en">undertaken by</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-der-der-fwd;UnderlyingValuation"/>
 		<rdfs:range rdf:resource="&fibo-der-der-fwd;CalculationAgent"/>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;valuationDate">
-		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasDate"/>
-		<rdfs:label xml:lang="en">valuation date</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardValuationTermsSet"/>
-		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Date"/>
-	</owl:ObjectProperty>
+	<owl:Class rdf:about="&fibo-fbc-fi-fi;Future">
+		<rdfs:subClassOf rdf:resource="&fibo-der-der-fwd;ForwardContract"/>
```

Comment:
I'm guessing that this class represents a specific held contract, acquired (possibly with others) on a specific date by a specific owner, rather than the exchange listing with the standardized terms. Need to have a property to link it to the latter.